src/fhir/testUtilities.tsx

```diff
@@ -111,17 +104,15 @@ const deleteAndCascadeDelete = async (identifiers: string[], resource: Retrievab
  * Contains hooks for displaying the modal and the fhir client context being set.
  * @constructor
  */
-export const TestReportForm: React.FC = () => {
+export const ContextAndModal: React.FC<{ children: ReactNode }> = (props) => {
   const client = FHIR.client(FHIR_URL);
 
   return (
     <>
       <div id="backdrop-hook"></div>
       <div id="modal-hook"></div>
       <div id="root"></div>
```

Comment:
Interesting. So I guess these need to be redeclared then even though the elements exist in the public index.html?

Reply:
Yeah, because we're only rendering specific components in the tests, rather than the entire application